store-and-retrieve-data/use-storagehub-sdk/end-to-end-storage-workflow.md

```diff
@@ -1,4 +1,104 @@
 ---
 title: End-to-End Storage Workflow
-description: Step-by-step tutorial on uploading a file to DataHaven and retrieving it from the network.
+description: Learn how to store and retrieve files on DataHaven with this step-by-step guide, from issuing a storage request to uploading and downloading data securely.
 ---
+
+# End-to-End Storage Workflow
+
+This guide walks you through the exact steps for storing a file on DataHaven and retrieving it. Follow each step in sequence; each guide linked below provides the detailed code you will run.
+
+## Workflow at a Glance
+
+For more details on each step, check out the respective guides. 
+
+Prerequisite: [Create a Bucket](/store-and-retrieve-data/use-storagehub-sdk/create-a-bucket/)
```

Comment:
Just curious why it was decided to include this as a prerequisite and not part of the guide?

Reply:
I think you're right, we should add the basic create a bucket functionality. I was originally apprehensive because the create a bucket guide is long, but that includes a lot of other verification, checking steps that are not really necessary in the create a bucket step.